database/fakeData.js

```diff
@@ -0,0 +1,42 @@
+// Condition: String,
```

Comment:
You probably can remove these commented out lines. Your schema will define what the types are right?

Reply:
sure I will do that . Thanks